### PR TITLE
Add name column for webhook and change dashboard messages for webhook

### LINKED ISSUE #1340: Add name column for webhook
Identifying webhooks by the URL might be misleading in some cases. So, it could be better to add an optional `name` column that can be used to better identify the webhook.

### LINKED ISSUE #1578: Improve dashboard messages when updating resource
At the moment, i'm working on the implementation of a "Name" field for the webhooks in Portus. #1340 
For testing purposes i changed the name of a webhook from "foobar" to "foobarhook".
The dashboard recognizes that, but it says "$USER updated "foobar" under the $NAMESPACE namespace."

So, it might be irritating to see the old webhook name under "Recent activities".
Therefore a special message like "$USER has updated the name of the webhook $OLD_WEBHOOK_NAME to $NEW_WEBHOOK_NAME".

### PR DESCRIPTION
**Pull request by lightlace**
Add a name column for webhook and change dashboard messages for webhook activities (e.g updating a webhook or creating a webhook).

Fix #1578 
Fix #1340

### CODE REVIEW

**Comment by lightlace** [comment]
@mssola @vitoravelino

File: db/migrate/20160411150441_create_webhooks.rb [hunk 1]
```diff
@@ -8,7 +8,6 @@ def change
       t.integer :request_method
       t.integer :content_type
       t.boolean :enabled, default: false
-
```
Comment (vitoravelino) [inline db/migrate/20160411150441_create_webhooks.rb]: Revert.

File: spec/controllers/webhooks_controller_spec.rb [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-
```
Comment (vitoravelino) [inline spec/controllers/webhooks_controller_spec.rb]: Revert

File: spec/factories/webhooks.rb [hunk 3]
```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-
```
Comment (vitoravelino) [inline spec/factories/webhooks.rb]: Revert

File: spec/models/webhook_spec.rb [hunk 4]
```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-
```
Comment (vitoravelino) [inline spec/models/webhook_spec.rb]: Revert

File: spec/controllers/webhooks_controller_spec.rb (hunk 2 above)
Comment (lightlace) [reply spec/controllers/webhooks_controller_spec.rb]: done.

File: db/migrate/20160411150441_create_webhooks.rb (hunk 1 above)
Comment (lightlace) [reply db/migrate/20160411150441_create_webhooks.rb]: done.

File: db/migrate/20180109114124_add_name_to_webhooks.rb~ [hunk 5]
```diff
@@ -0,0 +1,4 @@
+class AddNameToWebhooks < ActiveRecord::Migration
```
[the remark this answers is not in the document]
Comment (mssola) [reply db/migrate/20180109114124_add_name_to_webhooks.rb~]: You need to create a migration which adds the name. Moreover, this column should never be empty (hence I'd make it mandatory)

[the remark this answers is not in the document]
Comment (lavadew) [reply db/migrate/20180109114124_add_name_to_webhooks.rb~]: In app/views/webhooks/index.html.slim, the name field has the attribute "required" set to true, so the name cannot be empty. I thought that is enough.

[the remark this answers is not in the document]
Comment (vitoravelino) [reply db/migrate/20180109114124_add_name_to_webhooks.rb~]: It's a backup file ending with `~`. Please remove it.

File: db/migrate/20160411150441_create_webhooks.rb [hunk 6]
```diff
@@ -8,7 +8,7 @@ def change
...
+
```
Comment (vitoravelino) [inline db/migrate/20160411150441_create_webhooks.rb]: Remove whitespace.

File: db/migrate/20160411150441_create_webhooks.rb [hunk 7]
```diff
@@ -8,7 +8,6 @@ def change
       t.integer :request_method
       t.integer :content_type
       t.boolean :enabled, default: false
-
       t.timestamps null: false
```
Comment (vitoravelino) [inline db/migrate/20160411150441_create_webhooks.rb]: Nothing changed in this file. You can simply revert the file or create an empty line here to restore the previous state of the file and avoid this change.

File: db/migrate/20160411150441_create_webhooks.rb (hunk 6 above)
Comment (vitoravelino) [reply db/migrate/20160411150441_create_webhooks.rb]: File is still here...

**Comment by mssola** [comment]
The failure on travis does not look related to this PR (flaky test ? 😨)

**Comment by vitoravelino** [comment]
> The failure on travis does not look related to this PR (flaky test ? 😨)

I don't think so... Let's hope not.

**Pull request merged**